src/components/FeatureViews/Courses/CourseClasses.js

```diff
@@ -215,6 +224,17 @@ const CourseClasses = () => {
 
   const handleChange = (_, i) => setIndex(i);
 
+  const accountTypeCheck = (accountType, studentList, enrollmentArray) => {
+    switch(accountType) {
+      case "PARENT":
+        if(comparison(studentList, enrollmentArray)) {
+          return parentTabWithStudentEnrolledLabels
+        }
+        return [{ label: "About Course" }]
```

Comment:
what is this array? pull this out into its own explanatory variable. Is this just the default tab?